nephio-controllers/README.md

```diff
@@ -0,0 +1,21 @@
+# nephio
+
+## Description
+nephio controller
+
+## Usage
```

Comment:
I would just remove everything from here on, I think it can only lead to confusion.